### PR TITLE
Added custom icon component support to Input

### PR DESCRIPTION
**Pull request by FrederikBolding**
## Description
- Added custom icon component support to Input

### CODE REVIEW

**Comment by FrederikBolding** [comment]
Part of [ch3527]

File: src/atoms/Input/Input.story.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import Panel from '../Panel';
 import Input, { InputProps } from './Input';
 
 class ControlledInput extends Component<
-  InputProps &
+  InputProps<any> &
```
Comment (Mrtenz) [inline src/atoms/Input/Input.story.tsx]: Is it possible to make this more strict than `any` (here and in other places)

Comment (FrederikBolding) [reply src/atoms/Input/Input.story.tsx]: Maybe! I'm not the most proficient in TypeScript yet, so if anyone has suggestions, feel free!

**Comment by FrederikBolding** [comment]
@vilsbole I think everything should be fixed in the commit!

**Pull request merged**